R/make_report.R

```diff
@@ -14,6 +14,11 @@ if (getRversion() >= "2.15.1") {
 #' @param verbose Boolean that determines if progress output is displayed
 #' @param self_contained Boolean that determines if the final output should be a self contained html file
 #' @param open_browser Whether to open the created output in the browser
+#' @examples
+#'   # Run best with example(shinyloadtest_report, ask = FALSE)
```

Comment:
I would wrap this in `\donttest`

```
#' @examples
#'  \donttest{
#'   html_file <- sprintf("%s.html", tempfile("report"))
#'   message(sprintf("Generating report: %s", html_file))
#'   shinyloadtest_report(slt_demo_data_1, output = html_file)
#' }
```

Reply:
Why? Testing it seems to provide value.

Reply:
Resolving this for now since you approved so we can launch back to CRAN, but I do want to understand your perspective on this 👍

Reply:
It is an example that produces an artifact and could take over 5 seconds.  It is a great example of how to use it, but it seems invasive.  

Since it is runnable code (but shouldn't be run by default), I believe the example should be wrapped in `\donttest`.

(Where as any code in `\dontrun` I would claim to possibly never work.)

---------------

Counter argument to not use `\donttest`... 

It is an example that 
* doesn't take that long, 
* shows off the full functionality, and 
* will be cleaned up when the temp folder is naturally deleted.


------------------

I lean towards conservative examples to avoid CRAN problems... but they also asked for it.  Both approaches have merit, so I approved as is.  (Also, running examples are always better than non-running examples.)

Reply:
Looking through https://cran.r-project.org/doc/manuals/R-exts.html#index-_005cexamples , I do not see anything about **not** producing temp files.  

So as long as `R CMD check` doesn't complain about execution time, keep it in! (Sorry for the confusion.)

Reply:
Thanks for elaborating, makes total sense to me. After suffering this running example I'm leaning back toward `\donttest` because I'm having problems with rhub's pandoc version. I'll spend a little more time on it, but if I fail I think the example will need to become a `skip_on_cran` test.